src/services/pig.py

```
@@ -307,28 +308,12 @@ def handover_pig(
-
-            member_response = PigMemberResponse(
-                id=member.id,
-                ig_id=member.ig_id,
-                user_id=member.user_id,
-                created_at=member.created_at,
-                user=user_response,
-            )
-            res.append(member_response)
-
-        return res
+    def get_members(self, id: int) -> Sequence[PIGMember]:
```

Ruff (0.15.6)

[error] Function argument `id` is shadowing a Python builtin

(A002)